comfyui_pano_suite/nodes.py

```
@@ -189,27 +416,358 @@ def _hex_color_to_rgb01(value: str) -> np.ndarray:
+    return _normalize_image_batch_array(arr, allow_alpha=allow_alpha, fallback_shape=fallback_shape)
+
+
+def _repeat_mask_batch(mask_bw: np.ndarray, batch_size: int) -> np.ndarray:
+    src = np.clip(np.asarray(mask_bw, dtype=np.float32), 0.0, 1.0)
+    target_batch = max(1, int(batch_size))
+    if src.ndim == 3:
+        if int(src.shape[0]) == target_batch:
+            return src
+        if int(src.shape[0]) <= 0:
+            return np.zeros((target_batch, *src.shape[1:]), dtype=np.float32)
+        repeats = int(math.ceil(target_batch / float(src.shape[0])))
```

Ruff (0.15.10)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)